packages/common/src/scopeSupportFacets/typescript.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+import {
+  LanguageScopeSupportFacetMap,
+  ScopeSupportFacetLevel,
+} from "./scopeSupportFacets.types";
+
+const { supported } = ScopeSupportFacetLevel;
+
+export const typescriptScopeSupport: LanguageScopeSupportFacetMap = {
```

Comment:
This feels like a lie to say we only support these 5 things in TS. We should create a way to "import" support from another language, including the tests, the way we do for our `.scm` files. Shall I file an issue for that?

Reply:
Please do